mayastor/src/subsys/registration/mod.rs

```diff
@@ -0,0 +1,185 @@
+//! A Registration subsystem is used to keep control-plane in the loop
+//! about the lifecycle of mayastor instances.
+
+/// Module for grpc registration implementation
+pub mod registration_grpc;
+
+pub mod misc {
+    use strum_macros::{AsRefStr, ToString};
+
+    /// Error type which is returned for any operation
+    #[derive(Serialize, Deserialize, Clone, Debug)]
```

Comment:
Don't think it's worth adding ReplyError here since we're just converting from Status.

Reply:
Removed it.